src/Backend/Core/Js/backend.js

```diff
@@ -1974,8 +1974,18 @@ jsBackend.tableSequenceByDragAndDrop =
                         var module = (typeof $table.parents('table.jsDataGrid').data('module') == 'undefined') ? jsBackend.current.module : $table.parents('table.jsDataGrid').data('module').toString();
 
                         // fetch extra params
-                        if (typeof $table.parents('table.jsDataGrid').data('extra-params') != 'undefined') extraParams = $table.parents('table.jsDataGrid').data('extra-params');
-                        else extraParams = {};
+                        if (typeof $table.parents('table.jsDataGrid').data('extra-params') != 'undefined') {
+                            // we define extra params
+                            extraParams = $table.parents('table.jsDataGrid').data('extra-params');
+
+                            // we convert the unvalid {'key':'value'} to the valid {"key":"value"}
+                            extraParams = extraParams.replace(/'/g, '"');
```

Comment:
@WouterSioen That is not possible, since '"' breaks html.

For example:

You have the following in your Backend Action:
```php
// enable sorting
$this->dgPosts->enableSequenceByDragAndDrop();
$this->dgPosts->setAttributes(
    array(
        'class' => 'dataGrid sequenceByDragAndDrop',
        'data-extra-params' => "{'module':'Products', 'type':'" . $this->category['type'] . "', 'category_id':'" . $this->categoryId . "'}",
        'data-module' => 'Categories',
        'data-action' => 'SequenceForItems'
    )
);
```
> You can not use ", because this breaks the html. So you must use `.

In older jQuery versions this wasn't any problem.
But in the newer jQuery version I noticed that the json validation has become stricter.

**So we must convert ` to " in our javascript.** (=> which is exactly what this PR does).

===
SCREENSHOTS
**Before fix**
![schermafbeelding 2016-02-18 om 10 20 41](https://cloud.githubusercontent.com/assets/588616/13138774/4f9530ae-d629-11e5-8eff-a5aaa6af2f58.png)

**After fix**
![schermafbeelding 2016-02-18 om 10 20 26](https://cloud.githubusercontent.com/assets/588616/13138779/54fbecb8-d629-11e5-852e-d08ca5067fa3.png)